RoryMercury.json

```diff
@@ -0,0 +1,11 @@
+{
+	"type" : 0,
+	"english-name" : "Futaba Sakura",
+	"image" : "https://vignette.wikia.nocookie.net/gate-thus-the-jsdf-fought-there/images/f/fe/Rory.jpg/revision/latest?cb=20180112033407",
+	"japanese-name" : "ロリー·マーキュリ",
+	"year" : ?,
+	"month" : ?,
+	"day-of-month" : ?,
```

Comment:
not valid JSON, "?" is not a number. If you don't know the year, month, etc. then don't include it since it's an optional value. here's a good example https://github.com/yourWaifu/is-your-waifu-legal/blob/master/waifus/2b.json